src/components/table/table.scss

```diff
@@ -99,4 +90,42 @@
   color: $black;
 }
 
+.deprecated {
+  font-style: italic;
+  color: $dark-grey;
+
+  & td img {
+    opacity: 0.3;
+  }
+}
+
+.btn-deprecate-list {
+  background-color: $light-grey;
+  border-color: $light-grey;
+  &:hover {
+    color:white;
+  }
+}
+
+.btn-deprecate {
+  &:hover {
+    background-color: $red;
```

Comment:
You changed the names in **_colors.scss**, you need to change them here as well. 
Did you check this?